kolibri/plugins/learn/assets/src/views/ChannelThumbnail.vue

```diff
@@ -35,14 +36,38 @@
         default: false,
       },
     },
+    data() {
+      return {
+        resolvedThumbnail: this.thumbnail,
+      };
+    },
     computed: {
       thumbnailBackground() {
         return {
           backgroundColor: this.$themeTokens.surface,
-          backgroundImage: this.thumbnail ? `url('${this.thumbnail}')` : '',
+          backgroundImage: this.resolvedThumbnail ? `url('${this.resolvedThumbnail}')` : '',
```

Comment:
I think we can do a high-level fix here, where we just modify the `channels` on the library page.
Now it is affecting all the cards on the library page. I don't know where to make changes for that.